FORMULA

```diff
@@ -0,0 +1,9 @@
+name: uwsgi
+os: Debian, Ubuntu, RedHat, Fedora, CentOS, Arch
+os_family: Debian, RedHat, Arch
+version: 1.0.0
```

Comment:
We've been converting quite a few formulas in the last few months.  On our comparative spreadsheet, we've got a different version number to use:

```suggestion
version: 0.2.0
```